en/config_mc/mc_trajectory_tuning.md

```diff
@@ -12,37 +15,45 @@ The algorithm that generates those setpoints is called a "trajectory generator".
-Setpoint value ("higher level") tuning is used to specify the mapping between the demanded setpoint and the desired setpoint.
-Poorly tuned setpoint values cannot result in instability, but may result in either very jerky or very unresponsive reactions to setpoint changes.
+The *desired setpoint* passed to the P/PID controller is itself calculated from a *demanded setpoint* based on a stick position (in RC modes) or from a mission command.
+The demanded setpoint can change very quickly (e.g. if a user moves stick from zero to maximum value as a "step").
+Vehicle flight characteristics are better if the corresponding desired setpoint changes as a "ramp".
 
-> **Tip** The demanded setpoint can change very quickly (e.g. if a user moves stick from zero to maximum value as a "step").
-  Vehicle flight characteristics are better if the corresponding desired setpoint changes as a "ramp".
+*Setpoint value tuning* ("higher level tuning") is used to specify the mapping between the *demanded  and the *desired* setpoints - i.e. defining the "ramp" at which the desired setpoint follows the demanded setpoint.
 
-The setpoint-value tuning parameters can be split into two groups: tuning parameters for position mode and tuning parameters for mission mode.
-Some parameters will have an effect on both modes.
+> **Tip** Poorly tuned [P/PID Gains](../config_mc/pid_tuning_guide_multicopter.md) can lead to instability.
+  Poorly tuned *setpoint values* cannot result in instability, but may result in either very jerky or very unresponsive reactions to setpoint changes.
 
-## Definitions
 
-The position controller ([diagram here](https://dev.px4.io/master/en/flight_stack/controller_diagrams.html#multicopter-position-controller)) consists of an outer **P** position-control loop and an inner **PID** velocity-control loop.
-Depending on the control (flight) mode either both loops are active or just the velocity control loop.
+## Flight Modes Trajectory Support {#modes}
 
-For the remainder of this topic the term **position-control** represents the case where both loops are active while **velocity-control** refers to the case when only the velocity control loop is in use.
+[Mission mode](../flight_modes/mission.md) used the [Jerk-limited](../config_mc/mc_jerk_limited_type_trajectory.md) trajectory all the time.
 
-## Implementations
+[Position mode](../flight_modes/position_mc.md) supports all the [trajectory types](#trajectory_implementation) listed below.
+It uses the [Jerk-limited](../config_mc/mc_jerk_limited_type_trajectory.md) trajectory by default; the other types can be set using [MPC_POS_MODE](../advanced_config/parameter_reference.md#MPC_POS_MODE).
 
-Two different implementations are available for each mode and can be selected using the parameters [MPC_POS_MODE](../advanced_config/parameter_reference.md#MPC_POS_MODE) and [MPC_AUTO_MODE](../advanced_config/parameter_reference.md#MPC_AUTO_MODE).
+No other modes support trajectory tuning.
```

Comment:
Is this correct? I was surprised that altitude hold does not support the trajectory.

Reply:
The jerk-limited trajectory is also used for the vertical axis in altitude mode :)

Reply:
Altitude mode uses the type selected by `MPC_POS_MODE` just for the vertical component only.

Reply:
@MaEtUgR 

So just to be very clear, in both position mode and altitude mode the sticks control
- speed over ground in both directions
- rate or rotation above ground plane (yaw)
- speed of ascent/descent.

I thin you're are saying that in position mode, the jerk limiting is applied to all of these inputs. So a quick move on speed over ground stick will translate as a ramped demanded setpoint.

But in altitude mode this will only happen for the altitude stick. The other controls will default to the "position mode type 0 direct control type algorithm"?

Why is it done this way? i.e. why do we only choose to limit the jerkyness of the ascent/descent?

Reply:
I think there's a remaining misconception:
- In Position mode you run the position controller on all 3 axes and it's smoothened using the `MPC_POS_MODE` you choose.
- In Altitude mode the position controller is only ran against the altitude (world frame local z-axis). The altitude is controlled and smoothed exactly the same way as in position mode with the method specified by `MPC_POS_MODE`. The horizontal stick input is mapped to the acceleration or vehicle attitude. That's why the vehicle needs constant manual input to not drift away because there's no horizontal position or velocity control.

Yaw is the same for both, stick results in yaw rotational velocity and letting go of the stick locks the absolute yaw orientation.

Reply:
Thanks very much. I've clarified and merged.